mindmeld/components/_config.py

```diff
@@ -912,3 +918,102 @@ def get_nlp_config(app_path=None, config=None):
         pass
 
     return _get_default_nlp_config()
+
+
+def _get_default_regex(exclude_from_norm):
+    # List of regex's for matching and tokenizing when keep_special_chars=True
+    keep_special_regex_list = []
+
+    exception_chars = "\@\[\]\|\{\}'"  # noqa: W605
+
+    # fetches all currency symbols in unicode by iterating through the character set and
+    # selecting the currency symbols based on the unicode currency category 'Sc'
+    currency_symbols = u"".join(
```

Comment:
Is this a resource intensive process, ie how many iters happens here? If it is intensive, lets just make a list of current symbols and add it to constants.py

Reply:
It's a single iteration through the loop, so I think this setup should be fine (I've moved it to constants.py though)? It's forward looking as it will allow for any changes to the unicode currency structure to be automatically included in MindMeld without having to update the list manually.